GitUI/Theming/Win32ThemeDelegates.cs

```diff
@@ -13,7 +13,7 @@ namespace GitUI.Theming
     internal delegate int DrawThemeBackgroundDelegate(
         IntPtr htheme, IntPtr hdc,
         int partId, int stateId,
-        ref NativeMethods.RECT prect, ref NativeMethods.RECT pcliprect);
```

Comment:
Without this PR, I currently get an exception on line 200 or so
No investigation, any hints?

![image](https://user-images.githubusercontent.com/6248932/74199827-91381100-4c65-11ea-822f-db9b306158c2.png)

"   at GitUI.Theming.Win32ThemeHooks.DrawThemeBackgroundHook(IntPtr htheme, IntPtr hdc, Int32 partid, Int32 stateid, RECT& prect, RECT& pcliprect) in F:\\dev\\gc\\gitextensions_4\\GitUI\\Theming\\Win32ThemeHooks.cs:line 200\r\n   at System.Windows.Forms.UnsafeNativeMethods.CallWindowProc(IntPtr wndProc, IntPtr hWnd, Int32 msg, IntPtr wParam, IntPtr lParam)\r\n   at System.Windows.Forms.NativeWindow.DefWndProc(Message& m)\r\n   at System.Windows.Forms.Control.DefWndProc(Message& m)\r\n   at System.Windows.Forms.Control.WndProc(Message& m)\r\n   at System.Windows.Forms.ComboBox.WndProc(Message& m)\r\n   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)\r\n   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)\r\n   at System.Windows.Forms.NativeWindow.DebuggableCallback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)"

Reply:
The last commit of this PR fixes it. @RussKie did a cleanup of interop-related code, which changed RECT class used in theming to ref RECT struct. It causes NullReferenceException because theming API sends `null` to `pcliprect` variable, which does not work with ref struct. So I changed it back.

Reply:
Thanks once again